src/bundle/Resources/public/js/scripts/core/autogenerator.input.value.js

```diff
@@ -0,0 +1,56 @@
+(function (global, ibexa) {
+    class InputValueAutogenerator {
+        constructor(config = {}) {
+            this.sourceInput = config.sourceInput;
+            this.targetInput = config.targetInput;
+            this.whitespaceTextReplacer = config.whitespaceTextReplacer || '_';
```

Comment:
```suggestion
            this.whitespaceTextReplacer = config.whitespaceTextReplacer ?? '_';
```
because for `whitespaceTextReplacer` equal to `''` this would result in `_`.